hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/functional/TestBootstrapReadBase.java

```diff
@@ -103,6 +103,8 @@ protected Map<String, String> basicOptions() {
         options.put(HoodieWriteConfig.KEYGENERATOR_CLASS_NAME.key(), SimpleKeyGenerator.class.getName());
       } else {
         options.put(HoodieWriteConfig.KEYGENERATOR_CLASS_NAME.key(), ComplexKeyGenerator.class.getName());
+        // Disable complex key generator validation so that the writer can succeed
+        options.put(HoodieWriteConfig.ENABLE_COMPLEX_KEYGEN_VALIDATION.key(), "false");
```

Comment:
I don't think we need this for a fresh table of 1.1.

Reply:
I think test will fail currently but I think for new table you are likely right.